src/coreclr/jit/emit.cpp

```diff
@@ -1057,8 +1057,6 @@ void emitter::emitBegFN(bool hasFramePtr
 
     emitHasFramePtr = hasFramePtr;
 
-    emitMaxTmpSize = maxTmpSize;
```

Comment:
Can you also remove the `maxTmpSize` argument to `emitBegFN()` as well as the line:

```
 unsigned maxTmpSize = regSet.tmpGetTotalSize(); // This is precise after LSRA has pre-allocated the temps.
```
before the call to `emitBegFN`?

Reply:
Done!